src/test/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The " + getName() + " began its work.");
```

Comment:
The message in the 'doWork' method should be more informative and indicate that the 'Truck' is working right now. Consider including the type of machine in the message, for example: 'Truck ' + getName() + ' started its work.'